cli/package.json

```diff
@@ -52,6 +52,7 @@
   },
   "devDependencies": {
     "@prisma/client": "^4.11.0",
+    "@t3-oss/env-nextjs": "^0.1.0",
```

Comment:
why is this a lower version than in the template package.json?

Reply:
must have missed it...